docs/api-reference/next/image.md

```diff
@@ -234,6 +234,13 @@ Try it out:
 
 You can also [generate a solid color Data URL](https://png-pixel.com) to match the image.
 
+### rootMargin
+
+The IntersectionObserver interface's `rootMargin` property is a string with syntax similar to that of the CSS margin property.
+Defaults to `"200px"`.
+
+[Learn more](https://developer.mozilla.org/en-US/docs/Web/API/IntersectionObserver/rootMargin)
```

Comment:
The IntersectionObserver is really an implementation detail that we might remove in the future.

Furthermore, the name is very misleading.

Consider this example: `<Image src={img} rootMargin="100px" />`.

It could be misinterpreted as a CSS property.

A better name is probably `lazyMargin` or even `lazyBoundary`, but I also want feedback from @atcastle since it would be great to align the name here with any proposals for native img lazy loading boundaries.